create_pdfs.py

```diff
@@ -37,8 +37,12 @@ def parse_args(args=sys.argv[1:]):
     parser.add_argument('-i', '--issued-date', help='optional issue date')
     parser.add_argument(
         '-T',
-        '--assign-title',
-        help='optional title string for user, CME certs have a standard list with additional logic',
+        '--assign-designation',
```

Comment:
Since you're already making a (justified) backwards-incompatible change here,
this should be set to `--designation`.